nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/provenance/ui/provenance-event-listing/provenance-event-table/provenance-event-table.component.html

```diff
@@ -18,7 +18,9 @@
 <div class="provenance-event-table h-full">
     <div [class.hidden]="showLineage" class="h-full flex flex-col gap-y-2">
         <div class="flex flex-col">
-            <div class="value font-bold">Displaying {{ filteredCount }} of {{ totalCount }}</div>
+            <div [class.invisible]="!filterApplied" class="value font-bold">
+                Displaying {{ filteredCount }} of {{ totalCount }}
```

Comment:
This is still not very intuitive IMO. Maybe something like

"Filter matched {{filterCount}} of {{totalCount}}"

Since we still aren't displaying all of the matches if they span more than 1 page.